src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -213,6 +230,8 @@ def tree_onnx_graph_preprocessing(
         framework (str): The framework from which the ONNX model is generated.
             (options: 'xgboost', 'sklearn')
         expected_number_of_outputs (int): The expected number of outputs in the ONNX model.
+        use_fhe_sum (bool): This parameter is exclusively used to tree-based models.
+            It determines whether the sum of the trees' outputs is computed in FHE.
```

Comment:
I don't think we need to specify that the parameter is for tree based model since it's in the tree...py file. Also, what happens for decision tree models that have only 1 tree?

Reply:
Decision trees without the sum in FHE:
![image](https://github.com/zama-ai/concrete-ml/assets/14870263/50c8d880-4ca1-4353-8938-42b91067ba62)

Reply:
@jfrery @andrei-stoian-zama, we have a new node for decision tree with 1 tree. 
We are like 1.1 slower

Reply:
ok, no big deal, as long as it produces correct results

Reply:
I think that we shouldn't bother with that now, especially since the feature is currently disabled